packages/api/src/brevo-api/brevo-api-campaigns.service.ts

```diff
@@ -14,7 +15,11 @@ export class BrevoApiCampaignsService {
 
     constructor(@Inject(BREVO_MODULE_CONFIG) private readonly config: BrevoModuleConfig) {
         this.campaignsApi = new SibApiV3Sdk.EmailCampaignsApi();
-        this.campaignsApi.setApiKey(SibApiV3Sdk.EmailCampaignsApiApiKeys.apiKey, config.brevo.apiKey);
+    }
+
+    private setApiKey(scope: EmailCampaignScopeInterface): void {
+        const { apiKey } = this.config.brevo.getBrevoConfig(scope);
```

Comment:
I can't see why this method is available here since it's not defined in the `ConfigModule` (but only in `BrevoModule.register`). I guess I'm missing something here(?)